internal/backend/basicstation/backend.go

```diff
@@ -251,6 +255,16 @@ func (b *Backend) handleRouterInfo(r *http.Request, c *websocket.Conn) {
 		URI:    fmt.Sprintf("%s://%s/gateway/%s", b.scheme, r.Host, lorawan.EUI64(req.Router)),
 	}
 
+	if b.verifyCN && r.TLS != nil {
```

Comment:
I would add `&& len(r.TLS.PeerCertificates) > 0`, just to make sure that an empty `PeerCertificates` slice would not crash the LoRa Gateway Bridge.

Reply:
Ahh, of course. I was working on the assumption that verifyCN would only be set if there would be one, but that's a good way to lead to issues.